etc/helm/values/staging.yaml

```diff
@@ -0,0 +1,30 @@
+image:
```

Comment:
@lpaulmp I believe we're missing the original `values.yaml` file here. The idea is for that file to have ALL the variables listed. Sort of documentation. With either default values or "null" (indicating it needs to be defined when deploying). The staging.yaml and production.yaml should only override the parts they need to override.